geoviews/tests/plotting/mpl/test_chart.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+import xarray as xr
+import geoviews as gv
+
+from geoviews.element import WindBarbs
+
+from holoviews.tests.plotting.utils import ParamLogStream
+
+from geoviews import Store
+
+from test_plot import TestMPLPlot
+
+mpl_renderer = Store.renderers["matplotlib"]
+
+
+class TestWindBarbsPlot(TestMPLPlot):
+    def test_windbarbs(self):
+        x = np.linspace(-1, 1, 4)
+        X, Y = np.meshgrid(x, x)
+        U, V = 10 * X, 5 * Y
+
+        angle = np.pi / 2 - np.arctan2(-V, -U)
+        mag = np.hypot(U, V)
+
+        gv_barbs = WindBarbs((X, Y, angle, mag))
+
+        fig = gv.render(gv_barbs)
+        mpl_barbs = fig.axes[0].get_children()[0]
+        np.testing.assert_almost_equal(mpl_barbs.u.data, U.T.flatten())
+        np.testing.assert_almost_equal(mpl_barbs.v.data, V.T.flatten())
+
+    def test_windbarbs_dataset(self):
+        x = np.linspace(-1, 1, 4)
+        X, Y = np.meshgrid(x, x)
+        U, V = 10 * X, 1 * Y
+
+        angle = np.pi / 2 - np.arctan2(-V, -U)
+        mag = np.hypot(U, V)
+        ds = xr.Dataset(
+            {
+                "u": (["y", "x"], U),
+                "v": (["y", "x"], V),
+                "a": (["y", "x"], angle),
+                "m": (["y", "x"], mag),
+            },
+            coords={"x": x, "y": -x},
+        )
+
+        gv_barbs = gv.WindBarbs(ds, ["x", "y"], ["a", "m"])
+
+        fig = gv.render(gv_barbs)
+        mpl_barbs = fig.axes[0].get_children()[0]
+        np.testing.assert_almost_equal(mpl_barbs.u.data, U.T.flatten())
+        np.testing.assert_almost_equal(mpl_barbs.v.data, V.flatten())
```

Comment:
```suggestion
        np.testing.assert_almost_equal(mpl_barbs.v.data, V.T.flatten())
```

I think this is the reason for the failing test.

Reply:
Yeah thanks for catching that.